include/igl/copyleft/comiso/nrosy.cpp

```diff
@@ -311,9 +317,18 @@ void igl::copyleft::comiso::NRosyField::prepareSystemMatrix(const int N)
     if (!isFixed_j)
     {
       row = tag_t[j];
-      if (isFixed_i) b(row) += 2               * hard[i]; else T.push_back(Eigen::Triplet<double>(row,tag_t[i]  , -2             ));
-      if (isFixed_j) b(row) += -2              * hard[j]; else T.push_back(Eigen::Triplet<double>(row,tag_t[j] ,  2              ));
-      if (isFixed_p) b(row) += ((4 * igl::PI)/Nd) * p[eid] ; else T.push_back(Eigen::Triplet<double>(row,tag_p[eid],-((4 * igl::PI)/Nd)));
+      if (isFixed_i)
```

Comment:
More if statement braces.

Reply:
@fwilliams I do not agree. Adding brackets to single line ifs/loops take only the space. I am sure that Uncle Bob will agree with me. And probably already did in his book: Clean Code. Once you put them below the if() with a proper indention it is fine. I think. But well, if you will have this as a convention in the coding style guidelines then I will say let it be ;-)